packages/conjury/conjury.2.2/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+synopsis: "Conjury library for OMake"
+maintainer: "james woodyatt <jhw@conjury.org>"
+authors: "james woodyatt <jhw@conjury.org>"
+license: "BSD-2-Clause"
+tags: "org:conjury.org"
+homepage: "https://bitbucket.org/jhw/conjury/"
+bug-reports: "https://bitbucket.org/jhw/conjury/issues"
+depends: [
+  "ocaml" {with-test & native >= "4.08"}
+  "ocamlfind" {with-test & native & >= "1.7.3"}
+  "omake" {>= "0.10.3"}
```

Comment:
Should `omake` be marked as a build-only dependency? (I'm not sure because most build tools should but some (e.g., `dune`) should not.)